bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseSensorHandler.java

```diff
@@ -87,7 +88,7 @@ boolean validateConfig() {
             config.receivingEEPId.forEach(receivingEEP -> {
                 EEPType receivingEEPType = EEPType.getType(receivingEEP);
                 EEPType existingKey = receivingEEPTypes.putIfAbsent(receivingEEPType.getRORG(), receivingEEPType);
-                if (existingKey != null) {
+                if (!Objects.isNull(existingKey)) {
```

Comment:
From the documentation of `Objects.isNull`:
_Returns true if the provided reference is null otherwise returns false._

So this `IllegalArgumentException` is thrown when `existingKey` is not null

How is this check different over the current `existingKey != null`

Reply:
It's not different, this is only to get rid of the last (and related) compiler warning:
```
[WARNING] bundles\org.openhab.binding.enocean\src\main\java\org\openhab\binding\enocean\internal\handler\EnOceanBaseSensorHandler.java:[90,21] Redundant null check: The variable existingKey cannot be null at this location
```